### ISSUE TITLE
Search page facet checkboxes should have a more distinct highlight for keyboard navigation

### ISSUE THREAD

**Issue by carsonicator** [inveniosoftware/invenio-app-rdm#1631]
## Problem Description
During keyboard-only interactions over the left navigation menu, top-level facets with subfacets have a distinct blue box highlighting the icon:

<img width="288" alt="Screen Shot 2022-05-18 at 11 31 45 AM" src="https://user-images.githubusercontent.com/11506832/169122729-45ac6621-e88d-4e5d-aac7-a5f1201caccd.png">

Code:
`<button aria-label="show subfacets" class="ui icon button iconhold"><i aria-hidden="true" class="angle right icon icon"></i></button>`

However, the checkboxes of the subfacets have only a very faint blue highlight, which is difficult to see. A couple of examples:

<img width="291" alt="Screen Shot 2022-05-18 at 11 32 01 AM" src="https://user-images.githubusercontent.com/11506832/169126141-2ba3fb61-f71e-4d5c-89bb-ccfa333090b2.png">

<img width="229" alt="Screen Shot 2022-05-18 at 11 58 58 AM" src="https://user-images.githubusercontent.com/11506832/169134109-6f1c07c0-c9f2-4959-912a-629cd46989c5.png">

Code:
`<div class="ui checkbox"><input aria-describedby="article-count" class="hidden" id="article-facet-checkbox" readonly="" tabindex="0" type="checkbox" value="article"><label for="article-facet-checkbox">Articles</label></div>`

This is also true for menu facet items without subfacets:

<img width="231" alt="Screen Shot 2022-05-18 at 11 52 16 AM" src="https://user-images.githubusercontent.com/11506832/169128199-94181b39-b383-4ca3-8aa9-4bfe5c56cf95.png">

## Describe the solution you'd like
Keyboard-only navigation would be easier if the faint blue highlights mentioned above were replaced by higher line weight checkboxes that resembled the distinct blue box of the top-level facets mentioned above.
